src/pages/List.js

```diff
@@ -42,21 +57,26 @@ export default function List({ token }) {
             </section>
           ) : (
             <ul>
-              {listItem.docs.map((doc) => (
-                <li key={doc.id} className="checkbox-wrapper">
-                  <label>
-                    <input
-                      type="checkbox"
-                      defaultChecked={compareTimeStamps(
-                        doc.data().last_purchased,
-                      )}
-                      disabled={compareTimeStamps(doc.data().last_purchased)}
-                      onClick={(e) => markItemPurchased(e, doc.id)}
-                    />
-                    {doc.data().item_name}
-                  </label>
-                </li>
-              ))}
+              {listItem.docs
+                .filter(
+                  (doc) =>
+                    doc.data().item_name.includes(filter) || filter === '',
```

Comment:
Nice work! Very clean and works as expected! Two edge cases you may want to consider:
- accounting for extra spaces before/after
- account for different capitalization

Great job!

Reply:
Thank you for that! We will take what time we have to investigate the edge cases you suggested.

Reply:
Edge case suggestions have been addressed and added to code.